scripts/process_schemas.py

```diff
@@ -334,6 +350,12 @@ def run(self):
         sg = SchemaGenerator(self.version, directory, self.destinationFile)
         sg.write()
 
+    def run(self):
+        if self.sourceTar is None:
+            self.runWithDir(self.schemaDir)  # local mode
```

Comment:
This is confusing - we shouldn't pass an instance variable as a method parameter.